tests/conftest.py

```diff
@@ -73,6 +73,7 @@ def load_test_data_transactional(transactional_db, django_db_blocker):
     with django_db_blocker.unblock():
         call_command("loaddata", "integreat_cms/cms/fixtures/test_roles.json")
         call_command("loaddata", "integreat_cms/cms/fixtures/test_data.json")
+        call_command("loaddata", "integreat_cms/cms/fixtures/test_regions.json")
```

Comment:
```suggestion
```
I think loading an additional fixture for all test cases is a bit overkill if only one test case is using it. Instead, either load the fixture in the specific test case or generate the required objects on demand.